src/current/v22.2/take-full-and-incremental-backups.md

```diff
@@ -149,12 +149,20 @@ To view the available backup subdirectories, use [`SHOW BACKUPS`](show-backup.ht
 To take incremental backups, you need an [Enterprise license](enterprise-licensing.html).
 {{site.data.alerts.end}}
 
-If your cluster grows too large for nightly [full backups](#full-backups), you can take less frequent full backups (e.g., weekly) with nightly incremental backups. Incremental backups are storage efficient and faster than full backups for larger clusters.
-
-Incremental backups are smaller and faster to produce than full backups because they contain only the data that has changed since a base set of backups you specify (which must include one full backup, and can include many incremental backups). You can take incremental backups either as of a given timestamp or with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html).
+If your cluster grows too large for daily [full backups](#full-backups), you can take less frequent full backups (e.g., weekly) with daily incremental backups. Incremental backups are storage efficient and faster than full backups for larger clusters.
 
 If you are taking backups on a regular cadence, we recommend [creating a schedule](create-schedule-for-backup.html) for your backups.
 
+### Recommendations for incremental backup frequency
+
+Incremental backups form chains between full backups. Each incremental backup contains only the data that has changed since a base set of backups. This base set of backups must include one full backup and can include many incremental backups, which are smaller and faster to produce than full backups. You can take incremental backups either as of a given timestamp or with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html).
+
+Cockroach Labs recommends taking incremental backups every 10 minutes. CockroachDB supports up to 48 incremental backups between full backups. This may vary based on your specific use-case, so we recommend testing within your own environment and workloads. This can look like:
```

Comment:
None of the schedules you list below meet the recommendation of 10 minute. Also, if CRDB only supports 48 incrementals/day = 30 min/incremental, than how is it even possible to meet that recommendation without doing more than one full/day? I feel like I'm missing something or else the math doesn't add up?

Reply:
Also, that is a pretty extreme backup schedule (10 min), which would only ever be appropriate for critical production data (you certainly wouldn't want to wast the juice and storage to backup junk data every 10 min right?

Reply:
Perhaps it would make sense to remove the recommendation on 10 minutes? That was the recommendation given for v23.1 but for 22.2 it'd be hard for CockroachDB to get down to 10 minute intervals since it can only do 48 incrementals between fulls.